frontend/src/components/ui/DrawflowWrapper.jsx

```diff
@@ -202,6 +203,9 @@ export default function DrawflowWrapper() {
   const dropPipeline = (pipelineData) => {
     const pipelineJson = JSON.parse(pipelineData);
     const { specs, path } = pipelineJson;
+    const newId = generatePipelineId(specs);
+    specs.id = newId;
+    specs.history = newId;
```

Comment:
Is this required? I don't think it's part of the specs.

Reply:
### Explanation of Changes

1. **`const newId = generatePipelineId(specs);`**  
   This line was necessary to pass the pipeline name so that when the ID is generated, it results in something like: `OpenAI Agent Demo-cg6ga6hgygs3`.

2. **`specs.id = newId;`**  
   This is necessary to ensure that the pipeline has a new ID each time it is dragged and dropped.

3. **`specs.history = newId;`**  
   This line is not strictly necessary. I will move this out in the next update.

Reply:
Oh sorry, I was only asking about the `specs.history = newId;` but GitHub took the whole section of code instead.